### PR TITLE
populating HazardCategories

### PR DESCRIPTION
**Pull request by fvanderbiest**
These are required to insert the decision tree results into the datamart.
Please review, thanks.

### CODE REVIEW

File: thinkhazard_common/scripts/initializedb.py [hunk 1]
```diff
@@ -63,3 +64,13 @@ def populate_datamart(engine):
...
         DBSession.add(r)
+
+    # HazardCategory
+    hazardlevels = DBSession.query(HazardLevel)
+    for hazardtype in DBSession.query(HazardType):
+        for hazardlevel in hazardlevels:
+            r = HazardCategory()
+            r.hazardtype = hazardtype
+            r.hazardlevel = hazardlevel
+            r.general_recommendation = u'stay calm and relax'
```
Comment (pgiraud) [inline thinkhazard_common/scripts/initializedb.py]: In order to make it more relevant, can you please use the following:

``` python
r.general_recommendation = u'General recommendation for {} {}'.format(hazardtype.mnemonic, hazardlevel.mnemonic)
```

Comment (fvanderbiest) [inline thinkhazard_common/scripts/initializedb.py]: Sounds fair ;-)

**Comment by fvanderbiest** [comment]
Done.

**Comment by pgiraud** [comment]
Please merge.

**Pull request merged**